controllers/v1/observationsController.js

```diff
@@ -17,6 +17,7 @@ const FileStream = require(ROOT_PATH + '/generics/fileStream');
 const assessorsHelper = require(MODULES_BASE_PATH + '/entityAssessors/helper');
 const programsHelper = require(MODULES_BASE_PATH + '/programs/helper');
 const validateEntities = process.env.VALIDATE_ENTITIES ? process.env.VALIDATE_ENTITIES : 'OFF';
+const entityManagementService = require(ROOT_PATH + '/generics/services/entity-management');
```

Comment:
@borkarsaish65 do not directly import or use external services in the controller

Reply:
done

Reply:
@borkarsaish65 why we are still calling entityManagementService in controller?

Reply:
removed now